src/Web/WebMVC/WebMVC.csproj

```
@@ -31,11 +31,11 @@
     <PackageReference Include="Microsoft.AspNetCore.HealthChecks" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.3" />
```

The current recommended practice regarding the Microsoft.AspNetCore.App is  to use it without version, i.e.:

```suggestion
    <PackageReference Include="Microsoft.AspNetCore.App" />
```

---

done